src/Symfony/Component/Yaml/Parser.php

```diff
@@ -171,7 +171,7 @@ public function parse($value, $flags = 0)
                     $this->refs[$isRef] = end($data);
                 }
             } elseif (
-                self::preg_match('#^(?P<key>'.Inline::REGEX_QUOTED_STRING.'|(?:![^\s]+\s+)?[^ \'"\[\{!].*?) *\:(\s+(?P<value>.+?))?\s*$#u', $this->currentLine, $values)
+                self::preg_match('#^(?P<key>'.Inline::REGEX_QUOTED_STRING.'|(?:![^\s]++\s++)?[^ \'"\[\{!].*?) *\:(\s++(?P<value>.+))?$#u', $this->currentLine, $values)
```

Comment:
@GuilhemN the issue was the last part of the regexp, the related to the last `\s*` and the ungreedy quantifier just before.